mason_templates/mason-lock.json

```diff
@@ -0,0 +1 @@
+{"bricks":{"digit_entity":{"path":"/Users/ajil.oommen/Desktop/eGov/health/health-campaign-field-worker-app/mason_templates/digit_entity"},"freezed_bloc":{"path":"/Users/ajil.oommen/Desktop/eGov/health/health-campaign-field-worker-app/mason_templates/freezed_bloc"}}}
```

Comment:
will this be an issue if/when other devs pick this up? Can we have a relative path here instead?